client/src/App.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import logo from './logo.svg';
 import './App.css';
 
-function App() {
+function App(props: any) {
```

Comment:
not sure how to type drizzle. would it be
```
type props = { drizzle: Drizzle }
```

Reply:
If you have the types for drizzle already installed skip to the code. Otherwise, see if there are open source typings for drizzle and install them, if not you'll need to maintain your own. Something like https://github.com/JoshNavi/action-netlify-deploy/blob/master/src/types.d.ts

Then you can type the props, also I would recommend.

I would recommend ditching the function syntax here as well:

```tsx
type Props = { drizzle: Drizzle };

const App: React.FC<Props> = ({ drizzle }) => {
  return whatever;
}
```